Chapter4/ControlStatementPractise/src/IfStatementPractise.java

```diff
@@ -0,0 +1,9 @@
+
+public class IfStatementPractise {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+
+	}
```

Comment:
@aamir717 Are you going to add some code in here? If so, can you please send me a separate pull request for that?